Source/Instrument Addon.lvproj

```diff
@@ -920,19 +904,19 @@
 				<Property Name="Bld_excludedDirectory[5]" Type="Path">user.lib</Property>
 				<Property Name="Bld_excludedDirectory[5].pathType" Type="Str">relativeToAppDir</Property>
 				<Property Name="Bld_excludedDirectoryCount" Type="Int">6</Property>
-				<Property Name="Bld_localDestDir" Type="Path">../Built/Scripting API</Property>
+				<Property Name="Bld_localDestDir" Type="Path">../Built/Custom Devices/Scripting API</Property>

```

Comment:
This and all related Scripting API Build assets need to go into "/Built/Custom Devices/Instrument Addon/Scripting API"